source/_patterns/02-organisms/components/content-aside.mustache

```diff
@@ -3,15 +3,15 @@
     <div class="content-aside__first-column">
       {{#status}}
         <div class="status-wrapper">
-          <h5 class="status-title{{#titleLength}} status-title__{{titleLength}}{{/titleLength}}">{{status.title}}</h5>
+          <h5 class="status-title{{#titleLength}} status-title__{{titleLength}}{{/titleLength}}">{{title}}</h5>
```

Comment:
Is this anything to do with the tab menu build?

Reply:
No, it is not related.